src/MSBuild/LiveLogger/MessageNode.cs

```diff
@@ -68,7 +69,7 @@ public string ToANSIString()
                         ANSIBuilder.Formatting.ForegroundColor.Red)}";
                 case MessageType.HighPriorityMessage:
                 default:
-                    return $"ℹ️ {ANSIBuilder.Formatting.Italic(Message)}";
+                    return $"ℹ️ Message {Code}: {ANSIBuilder.Formatting.Italic(Message)}";
```

Comment:
Changing this unconditionally means every message now has a colon in it:

![image](https://user-images.githubusercontent.com/3347530/216624942-f43694eb-3390-4c0e-b480-71bd0a1478c4.png)

I think it'd be better to add the code+colon only if a code exists.

Reply:
Oh I agree. Should it still say message and then a colon after that for those that don't have codes?

Reply:
IMO saying "message" is unnecessary; we're indicating that through formatting and the ℹ️